app/assets/stylesheets/charts.scss

```diff
@@ -62,6 +62,12 @@
   margin: 0px auto 27px auto;
 }
 
+.statistics-pie-placeholder{
+  margin: 12px auto;
+  width: 240px;
+  height: 268px;
+}
+
 .statistics-pie{
```

Comment:
@tbrisker there is some jquery hook that uses that class, it was my first attempt but it broke something else.